app/src/main/res/drawable/simple_dark_grey_gradient.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you please add the Mozilla licensing header?

```
<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
```

Reply:
(This looks good other than that 😄 )

Reply:
Added!